src/geom/transformFactory.cc

```diff
@@ -124,7 +124,7 @@ ast::PolyMap makeOneDDistortion(std::vector<double> const &coeffs) {
         polyCoeffs[i - 1][2] = i;
     }
 
-    return ast::PolyMap(polyCoeffs, 1, "IterInverse=1, TolInverse=1e-8, NIterInverse=20");
+    return ast::PolyMap(polyCoeffs, 1, "IterInverse=1, TolInverse=1e-8, NIterInverse=30");
```

Comment:
I assume this runs 30 steps every time it computes the inverse? How much slower is it?

Reply:
`NIterInverse` is an upper limit. For most transforms the solution will reach the tolerance specified by `TolInverse` long before it hits the limit on iterations and stop. However, our unit tests have found several examples where 20 is not enough and so the results are silently lousy. In each such case 30 has been more than enough. It's not a systematic approach, but 30 is clearly better than 20 for accuracy.